<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Răzvan V. Chereji</title>
<link href='https://fonts.googleapis.com/css?family=Palanquin:100' rel='stylesheet' type='text/css'>
<style type="text/css">
body {
	font-family: 'Palanquin', sans-serif;
	font-size: 24px;
	line-height: 1.5;
}
.header_text {
	font-family: 'Palanquin', sans-serif;
	font-size: 36px;
	font-weight: bold;
}
.email_image {
	vertical-align: middle;
}	
.text1 .main_header_text {
	font-family: 'Palanquin', sans-serif;
	font-size: 45px;
}
.text1 .header_text_small {
	font-family: 'Palanquin', sans-serif;
	font-size: 24px;
}
::selection {
    color: red; 
    background: yellow;
}

#wrapper {
    max-width: 1280px;
    margin: 0 auto;
}
</style>
</head>

<body>
<div id="wrapper">
<div class="text1">
  <p><img src="Razvan.jpg" height="300" alt="image" align="left" style="display:inline;margin:18px 30px 0px 0px;"/>
    <span class="main_header_text"><font color = "Brown">R&#x00103;zvan V. Chereji</font></span><br>
After completing my PhD and postdoc, studying statistical mechanics and computational biology, I decided to try new challenges and to change my career path. I like to learn new things, to keep growing, and in October 2019 I moved to Princeton to do research in quantitative finance.
</p>

<p><span class="header_text"><font color = "Brown">Curriculum Vitae</font></span><br>
You can download my CV from <a href="http://rchereji.github.io/cv.pdf">here</a>.</p>

<span class="header_text"><font color = "Brown">Publications</font></span><br>
(Please see my <a href="https://scholar.google.com/citations?user=DCuOYggAAAAJ&hl=en">Google Scholar profile</a> or search my name on <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Chereji+RV">PubMed</a> for the most up-to-date list of my publications.)<br>
<div class="csl-bib-body" style="line-height: 1.35; margin-left: 2em; text-indent:-2em;">
  <div class="csl-entry">26. Clark, S., <strong>Chereji, R.V.</strong>, Lee, P., Fields, R.D., Clark, D.J., 2019. Differential nucleosome spacing in neurons and glia, Neurosci Lett. (2019), 134559. <a href="https://doi.org/10.1016/j.neulet.2019.134559">https://doi.org/10.1016/j.neulet.2019.134559</a></div>
  <div class="csl-entry">25. <strong>Chereji, R.V.</strong>, Bryson, T.D., Henikoff, S., 2019. Quantitative MNase-seq accurately maps nucleosome occupancy levels, Genome Biol. 20(1), 1-18. <a href="https://doi.org/10.1186/s13059-019-1815-z">https://doi.org/10.1186/s13059-019-1815-z</a></div>
  <div class="csl-entry">24. <strong>Chereji, R.V.*</strong>, Eriksson, P.R.<strong>*</strong>, Ocampo, J.<strong>*</strong>, Prajapati, H.K., Clark D.J., 2019. Accessibility of promoter DNA is not the primary determinant of chromatin-mediated gene regulation, Genome Res. gr-249326. <a href="https://doi.org/10.1101/gr.249326.119">https://doi.org/10.1101/gr.249326.119</a></div>
  <div class="csl-entry">23. Ocampo, J.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Eriksson, P.R., Clark, D.J., 2019. Contrasting roles of the RSC and ISW1/CHD1 chromatin remodelers in RNA polymerase II elongation and termination. Genome Res. 29, 407–417. <a href="https://doi.org/10.1101/gr.242032.118">https://doi.org/10.1101/gr.242032.118</a></div>
  <div class="csl-entry">22. Hamdani, O., Dhillon, N., Hsieh, T.-H.S., Fujita, T., Ocampo, J., Kirkland, J.G., Lawrimore, J., Kobayashi, T.J., Friedman, B., Fulton, D., Wu, K.Y., <strong>Chereji, R.V.</strong>, Oki, M., Bloom, K., Clark, D.J., Rando, O.J., Kamakaka, R.T., 2019. Transfer RNA Genes Affect Chromosome Structure and Function via Local Effects. Mol. Cell Biol. <a href="https://doi.org/10.1128/MCB.00432-18">https://doi.org/10.1128/MCB.00432-18</a></div>
  <div class="csl-entry">21. Chang, H.-W., Valieva, M.E., Safina, A., <strong>Chereji, R.V.</strong>, Wang, J., Kulaeva, O.I., Morozov, A.V., Kirpichnikov, M.P., Feofanov, A.V., Gurova, K.V., Studitsky, V.M., 2018. Mechanism of FACT removal from transcribed genes by anticancer drugs curaxins. Science Advances 4, eaav2131. <a href="https://doi.org/10.1126/sciadv.aav2131">https://doi.org/10.1126/sciadv.aav2131</a></div>
  <div class="csl-entry">20. Mehta, G.D., Ball, D.A., Eriksson, P.R., <strong>Chereji, R.V.</strong>, Clark, D.J., McNally, J.G., Karpova, T.S., 2018. Single-Molecule Analysis Reveals Linked Cycles of RSC Chromatin Remodeling and Ace1p Transcription Factor Binding in Yeast. Mol. Cell 72, 875-887.e9. <a href="https://doi.org/10.1016/j.molcel.2018.09.009">https://doi.org/10.1016/j.molcel.2018.09.009</a></div>
  <div class="csl-entry">19. Rawal, Y.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Qiu, H., Ananthakrishnan, S., Govind, C.K., Clark, D.J., Hinnebusch, A.G., 2018a. SWI/SNF and RSC cooperate to reposition and evict promoter nucleosomes at highly expressed genes in yeast. Genes Dev. 32, 695–710. <a href="https://doi.org/10.1101/gad.312850.118">https://doi.org/10.1101/gad.312850.118</a></div>
  <div class="csl-entry">18. Ouda, R., Sarai, N., Nehru, V., Patel, M.C., Debrosse, M., Bachu, M., <strong>Chereji, R.V.</strong>, Eriksson, P.R., Clark, D.J., Ozato, K., 2018. SPT6 interacts with NSD2 and facilitates interferon-induced transcription. FEBS Letters 592, 1681–1692. <a href="https://doi.org/10.1002/1873-3468.13069">https://doi.org/10.1002/1873-3468.13069</a></div>
  <div class="csl-entry">17. <strong>Chereji, R.V.</strong>, Clark, D.J., 2018. Major Determinants of Nucleosome Positioning. Biophys. J. 114, 2279–2289. <a href="https://doi.org/10.1016/j.bpj.2018.03.015">https://doi.org/10.1016/j.bpj.2018.03.015</a></div>
  <div class="csl-entry">16. Rawal, Y.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Valabhoju, V., Qiu, H., Ocampo, J., Clark, D.J., Hinnebusch, A.G., 2018b. Gcn4 Binding in Coding Regions Can Activate Internal and Canonical 5' Promoters in Yeast. Molecular Cell 70, 297-311.e4. <a href="https://doi.org/10.1016/j.molcel.2018.03.007">https://doi.org/10.1016/j.molcel.2018.03.007</a></div>
  <div class="csl-entry">15. <strong>Chereji, R.V.*</strong>, Ramachandran, S.<strong>*</strong>, Bryson, T.D., Henikoff, S., 2018. Precise genome-wide mapping of single nucleosomes and linkers in vivo. Genome Biol. 19, 19. <a href="https://doi.org/10.1186/s13059-018-1398-0">https://doi.org/10.1186/s13059-018-1398-0</a></div>
  <div class="csl-entry">14. Johnson, T.A.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Stavreva, D.A., Morris, S.A., Hager, G.L., Clark, D.J., 2018. Conventional and pioneer modes of glucocorticoid receptor interaction with enhancer chromatin in vivo. Nucleic Acids Res. 46, 203–214. <a href="https://doi.org/10.1093/nar/gkx1044">https://doi.org/10.1093/nar/gkx1044</a></div>
  <div class="csl-entry">13. <strong>Chereji, R.V.*</strong>, Bharatula, V.<strong>*</strong>, Elfving, N., Blomberg, J., Larsson, M., Morozov, A.V., Broach, J.R., Björklund, S., 2017a. Mediator binds to boundaries of chromosomal interaction domains and to proteins involved in DNA looping, RNA metabolism, chromatin remodeling, and actin assembly. Nucleic Acids Res. 45, 8806–8821. <a href="https://doi.org/10.1093/nar/gkx491">https://doi.org/10.1093/nar/gkx491</a></div>
  <div class="csl-entry">12. <strong>Chereji, R.V.*</strong>, Ocampo, J.<strong>*</strong>, Clark, D.J., 2017b. MNase-Sensitive Complexes in Yeast: Nucleosomes and Non-histone Barriers. Mol. Cell 65, 565-577.e3. <a href="https://doi.org/10.1016/j.molcel.2016.12.009">https://doi.org/10.1016/j.molcel.2016.12.009</a></div>
  <div class="csl-entry">11. Ocampo, J.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Eriksson, P.R., Clark, D.J., 2016. The ISW1 and CHD1 ATP-dependent chromatin remodelers compete to set nucleosome spacing in vivo. Nucleic Acids Res. 44, 4625–4635. <a href="https://doi.org/10.1093/nar/gkw068">https://doi.org/10.1093/nar/gkw068</a></div>
  <div class="csl-entry">10. Qiu, H.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Hu, C., Cole, H.A., Rawal, Y., Clark, D.J., Hinnebusch, A.G., 2016. Genome-wide cooperation by HAT Gcn5, remodeler SWI/SNF, and chaperone Ydj1 in promoter nucleosome eviction and transcriptional activation. Genome Res. 26, 211–225. <a href="https://doi.org/10.1101/gr.196337.115">https://doi.org/10.1101/gr.196337.115</a></div>
  <div class="csl-entry">9. <strong>Chereji, R.V.*</strong>, Kan, T.-W.<strong>*</strong>, Grudniewska, M.K., Romashchenko, A.V., Berezikov, E., Zhimulev, I.F., Guryev, V., Morozov, A.V., Moshkin, Y.M., 2016. Genome-wide profiling of nucleosome sensitivity and chromatin accessibility in Drosophila melanogaster. Nucleic Acids Res. 44, 1036–1051. <a href="https://doi.org/10.1093/nar/gkv978">https://doi.org/10.1093/nar/gkv978</a></div>
  <div class="csl-entry">8. <strong>Chereji, R.V.</strong>, Morozov, A.V., 2015. Functional roles of nucleosome stability and dynamics. Brief. Funct. Genomics 14, 50–60. <a href="https://doi.org/10.1093/bfgp/elu038">https://doi.org/10.1093/bfgp/elu038</a></div>
  <div class="csl-entry">7. Cole, H.A., Ocampo, J., Iben, J.R., <strong>Chereji, R.V.</strong>, Clark, D.J., 2014. Heavy transcription of yeast genes correlates with differential loss of histone H2B relative to H4 and queued RNA polymerases. Nucleic Acids Res. 42, 12512–12522. <a href="https://doi.org/10.1093/nar/gku1013">https://doi.org/10.1093/nar/gku1013</a></div>
  <div class="csl-entry">6. Ganguli, D.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Iben, J.R., Cole, H.A., Clark, D.J., 2014. RSC-dependent constructive and destructive interference between opposing arrays of phased nucleosomes in yeast. Genome Res. 24, 1637–1649. <a href="https://doi.org/10.1101/gr.177014.114">https://doi.org/10.1101/gr.177014.114</a></div>
  <div class="csl-entry">5. <strong>Chereji, R.V.</strong>, Morozov, A.V., 2014. Ubiquitous nucleosome crowding in the yeast genome. Proceedings of the National Academy of Sciences USA 111, 5236–5241. <a href="https://doi.org/10.1073/pnas.1321001111">https://doi.org/10.1073/pnas.1321001111</a></div>
  <div class="csl-entry">4. Elfving, N.<strong>*</strong>, <strong>Chereji, R.V.*</strong>, Bharatula, V., Björklund, S., Morozov, A.V., Broach, J.R., 2014. A dynamic interplay of nucleosome and Msn2 binding regulates kinetics of gene activation and repression following stress. Nucleic Acids Res. 42, 5468–5482. <a href="https://doi.org/10.1093/nar/gku176">https://doi.org/10.1093/nar/gku176</a></div>
  <div class="csl-entry">3. Petrenko, N., <strong>Chereji, R.V.</strong>, McClean, M.N., Morozov, A.V., Broach, J.R., 2013. Noise and interlocking signaling pathways promote distinct transcription factor dynamics in response to different stresses. Mol Biol Cell 24, 2045–2057. <a href="https://doi.org/10.1091/mbc.e12-12-0870">https://doi.org/10.1091/mbc.e12-12-0870</a></div>
  <div class="csl-entry">2. <strong>Chereji, R.V.</strong>, Morozov, A.V., 2011. Statistical Mechanics of Nucleosomes Constrained by Higher-Order Chromatin Structure. J. Stat. Phys. 144, 379–404. <a href="https://doi.org/10.1007/s10955-011-0214-y">https://doi.org/10.1007/s10955-011-0214-y</a></div>
  <div class="csl-entry">1. <strong>Chereji, R.V.</strong>, Tolkunov, D., Locke, G., Morozov, A.V., 2011. Statistical mechanics of nucleosome ordering by chromatin-structure-induced two-body interactions. Phys. Rev. E 83, 050903. <a href="https://doi.org/10.1103/PhysRevE.83.050903">https://doi.org/10.1103/PhysRevE.83.050903</a></div>
  <div class="csl-entry">
    <p><strong>* </strong>These authors contributed equally</p>
  </div>
</div>
</div>
</div>
</body>
</html>
